chrome/content/zotero/collectionTree.jsx

```diff
@@ -826,6 +823,11 @@ var CollectionTree = class CollectionTree extends LibraryTree {
 				}
 			}
 		}
+		else if (action == 'refresh' && type == 'trash') {
+			// We need to update the trash's status (full or empty), and if empty,
+			// the row might be removed
```

Comment:
Why would the row be removed? Do we ever not show the trash? (Possible we should hide it if it's empty rather than adding an extra row for every group, but that's another question…)

Reply:
Yes, amazingly! I only learned that that’s an option while working on this PR. Pref is read here:

https://github.com/zotero/zotero/blob/6ca854a018e8bfe4251fbf42610276c441b5d943/chrome/content/zotero/collectionTree.jsx#L2311

Reply:
Huh! Who knew? (Certainly not me, who added that pref.)